ukat/data/tests/test_fetch.py

```diff
@@ -268,3 +268,41 @@ def test_philips_tsnr(self):
         assert isinstance(affine, np.ndarray)
         assert len(np.shape(data)) == 4
         assert np.shape(affine) == (4, 4)
+
+    def test_philips_pc_left(self):
```

Comment:
Same here, if it could be in alphabetical order (by sequence) that would be great